popover bug

test failing:

  ● Test suite failed to run

    libs/core/popover/popover-body/popover-body.component.spec.ts:14:38 - error TS2304: Cannot find name 'OverlayModule'.

    14             imports: [PopoverModule, OverlayModule, A11yModule]
                                            ~~~~~~~~~~~~~

 PASS   core-popover  libs/core/popover/popover-body/popover-body-directives/popover-body-header.directive.spec.ts
 PASS   core-popover  libs/core/popover/popover-control/popover-control.component.spec.ts
 PASS   core-popover  libs/core/popover/popover-body/popover-body-directives/popover-body-footer.directive.spec.ts

--- a/libs/core/popover/popover-body/popover-body.component.spec.ts
+++ b/libs/core/popover/popover-body/popover-body.component.spec.ts
@@ -11,7 +11,7 @@ describe('PopoverBodyComponent', () => {
 
     beforeEach(waitForAsync(() => {
         TestBed.configureTestingModule({
-            imports: [PopoverModule, OverlayModule, A11yModule]
+            imports: [PopoverModule, A11yModule]
         }).compileComponents();
     }));
 

